Essentials/src/com/earth2me/essentials/EssentialsConf.java

```diff
@@ -125,20 +111,20 @@ public synchronized void load() {
                         ByteBuffer resize = ByteBuffer.allocate(buffer.capacity() + length - buffer.remaining());
                         int resizePosition = buffer.position();
                         // Fix builds compiled against Java 9+ breaking on Java 8
-                        ((Buffer) buffer).rewind();
+                        buffer.rewind();
```

Comment:
```suggestion
                        ((Buffer) buffer).rewind();
```

These casts are necessary, otherwise builds on Java 9+ won't run on Java 8 (some people ran into issues with this).